examples/apps/PHP/F2wits/manifest.js

```diff
@@ -8,8 +8,7 @@ F2_jsonpCallback_com_openf2_examples_php_f2wits({
 	],	 
 	"apps":[{
 			"html":[
-				'<div class="f2-app-view" data-f2-view="home"></div>',
-				'<div class="f2-app-view hide" data-f2-view="about"><p>App version 1.0. Designed for F2 1.0</p></div>'
+				'<div class="f2-app-view" data-f2-view="home"></div>'
```

Comment:
At this point do you guys see any reason to keep any mention of `f2-app-view` and `data-f2-view` around?  Those aren't supported in V2 so could be cleaned up as well?